UI tweaks per material guidelines

vvimjam opened the pull request:
##What does this PR do
- Slightly modifies Main list layout per material guidelines
- Adds margin to Filtering type icon parent
- Adds app icon on click listner to initiate "app detail view" intent 

##Testing
This PR has been tested in Pixel 3 & Pixel C table in both potrait & landscape views.

vvimjam committed:
Improved main list per material guidelines, minor UI tweaks in main layout & added onclick listner for app icon

ukanth reviewed (commented):
Thanks for the PR. Looks good to me. Can we add some effect to the divider ? something like below ?

<?xml version="1.0" encoding="utf-8"?>
<shape xmlns:android="http://schemas.android.com/apk/res/android" >

    <gradient
        android:angle="180"
        android:startColor="#00000000"
        android:centerColor="#FFF29C"
        android:endColor="#00000000"/>

</shape>

to main.xml ListView ? 

android:divider="@drawable/list_divider"
android:dividerHeight="1px"

vvimjam committed:
Fixed app package text crop issue in app detail, Fixed invisible checkbox color in black theme, imroved checkbox color in dark theme, new main list divider

vvimjam committed:
Upstream beta branch sync

vvimjam commented:
Additional changes:

1. List view divider: Apologies, the gradient dividier was good but it felt out of place so i utilized material style divider. Code for gradient divider exists in /drawables/list_divider. Feel free to uncomment that and change the ListView divider height to 1px in main.xml.

2. Fixed app_package text view cut off bug: Open any app detailed view whose package name is too long and it would cutoff.

3. Fixed checkbox visibility in Amoled black theme: Checkbox was not visible in amoled black theme, changed the color to gray so that it is visible & this also improved visibility in default black theme.

ukanth commented:
Merged. I have updated line separator using gradient. Based on the feedback  from users I might revert or keep. Thanks.
